docs/admin/kubeadm.md

```diff
@@ -362,6 +406,29 @@ For the gory details on how the tokens are implemented (including managing them
 outside of kubeadm) see the [Bootstrap Token
 docs](/docs/admin/bootstrap-tokens/).
 
+## Running kubeadm without an internet connection
```

Comment:
No action to be taken but I hate that we have this here.  I'd much rather have `kubeadm` provide a way to dump this itself.